src/renderer/pages/Settings/Networks/Networks.tsx

```diff
@@ -201,7 +202,11 @@ export const Networks = () => {
           )}
         </NetworkList>
 
-        <NetworkList isDefaultOpen title={t('settings.networks.activeNetworksLabel')} networkList={activeNetworks}>
+        <NetworkList
+          query={filterQuery}
```

Comment:
Unless I missed something, there's no need to pass this as prop here, it can be fetched directly in the `NetworkList` component with `useUnit`

Reply:
`NetworkList` and `networkFilterModel` are both features and cannot be imported one inside another

Reply:
Re-reading the FSD doc, I understand what you mean thanks.